endpointmanager/pkg/fetcher/fetcher.go

```diff
@@ -94,6 +94,18 @@ func GetListOfEndpointsKnownSource(rawendpts []byte, source string, listURL stri
 			return result, fmt.Errorf("lantern list not given in Lantern format: %s", err)
 		}
 		result = LanternList{}.GetEndpoints(lanternList, listURL)
+	} else if source == "CareEvolution" {
```

Comment:
Similar to the Epic list update, we don't need to add separate sources for CareEvolution and 1Up since they're both processed the same way as Lantern